core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -27,8 +27,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-<l:layout permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%System}" type="one-column">
```

Comment:
One issue with the current approach is that there can be a mismatch between the page type and it's contents - on the `Configure` page it includes a sidebar however it isn't displayed due to the type being set to `one-column`. The inverse could be true as well where the page type is `two-column` but a side panel hasn't been provided.